PWGEM/Dilepton/Tasks/emEfficiencyEE.cxx

```diff
@@ -716,7 +716,14 @@ struct AnalysisTrackSelection {
       VarManager::FillTrackMC(groupedMCTracks, mctrack);
       int isig = 0;
       for (auto sig = fMCSignals.begin(); sig != fMCSignals.end(); sig++, isig++) {
-        if ((*sig).CheckSignal<TTracksMC>(true, groupedMCTracks, mctrack)) {
+        bool checked = false;
+        if constexpr (soa::is_soa_filtered_v<TTracksMC>) {
+          auto mctrack_raw = groupedMCTracks.rawIteratorAt(mctrack.globalIndex());
+          checked = (*sig).CheckSignal(false, mctrack_raw);
```

Comment:
I think in this case `checked = (*sig).CheckSignal(true, mctrack_raw);` should be used. 
Same in the line (724) below.